src/index.html

```diff
@@ -2,16 +2,218 @@
+          name = "Age" 
+          max="100"
+          min="0"
+          value = "12"
+          rows="2"
+          required
+          >
+        </div>
+
+        <div class="form-field">
+          <label>Full data of birth:
+          </label>
+          <input 
+          type = "date" 
+          name = "data" 
+          placeholder = "mm/ dd/ yyyy"
+          required
+          >
+        </div>
+
+        <div class="form-field">
+          <label>I accept the term of the agreement
+          </label>
+          <input 
+          type= "checkbox" 
+          name="Accept"
+          required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class = "container regictration">
+        <legend>Regictration</legend>
+          
+        <div class="form-field">
+          <label>E-mail:
+          </label>
+          <input 
+          type= "email" 
+          name="email"
+          required
+          placeholder = "email@example.com"
+          >
+        </div>
+
+        <div class="form-field">
+          <label>Password:
+          </label>
+          <input 
+            type = "password" 
+            name = "password"
+            required
+            minlength="8"
+            maxlength="30"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class = "container fact">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <div class="form-field">
+          <label>Do you lave cats?
+          </label>
+          
+          <label>
+            <input 
+              type = "radio" 
+              name = "cats"
+              value="Yes"
+            >
+          
+          Yes
+          </label>
+
+          <label>
+            <input 
+              type = "radio" 
+              name = "cats"
+              value="No"
+            >
+          
+          No
+          </label>
+        </div>
+        
+
+        <div class="form-field">
+          <label>What is yor favorite color?
+          </label>
+          <input 
+            type = "color" 
+            name = "color"
+          >
+        </div>
+
+        <div class="form-field">
+          <label>What time do you go to bed?
+          </label>
+          <input 
+            type = "time" 
+            name = "time"
+            placeholder = "--:--:-- --"
+          >
+        </div>
+
+        <div class="form-field">
+          <label>
+            What are your favorite brand of cars?
+          </label>
+          <select name = "cars" multiple>
+            <option value = "BMW">BMW</option>
+            <option value = "Audi">Audi</option>
+            <option value = "Lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="form-field">
+          <label>How do you rate our work?
+          </label>
+          <input 
+            type = "range" 
+            name = "rate"
+            value="0"
+          >
+        </div>
+
+      </fieldset>
```

Comment:
Don't add line breaks after the last child element. Should be fixed in all places
```suggestion
        </div>
      </fieldset>
```
```suggestion
        </div>
      </fieldset>
```